front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/pipeline/Pipeline.java

```diff
@@ -14,27 +14,21 @@
  * limitations under the License.
  *
  */
-package com.netflix.spinnaker.front50.model.pipeline;
+package com.netflix.spinnaker.front50.api.model.pipeline;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.common.base.Strings;
-import com.netflix.spinnaker.front50.model.Timestamped;
+import com.netflix.spinnaker.front50.api.model.Timestamped;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
 
 public class Pipeline extends HashMap<String, Object> implements Timestamped {
 
-  private static ObjectMapper MAPPER = new ObjectMapper();
   public static final String TYPE_TEMPLATED = "templatedPipeline";
 
-  @JsonIgnore
```

Comment:
JsonIgnore annotations removed here and moved to JsonIgnoreProperties annotation `PipelineMixins` class: https://github.com/mochacat/front50/pull/1/files#diff-7ba0d154a77fff1a603cb0e44f72b737bb5654280c48ebe32a36179d0cedd2d2